### PR TITLE
JumpStart: llama 2 chat finetuning

### PR DESCRIPTION
**Pull request by vivekmadan2**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [ ] I have tested my notebook(s) and ensured it runs end-to-end
- [ ] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: introduction_to_amazon_algorithms/jumpstart-foundation-models/llama-2-chat-completion.ipynb [hunk 1]
```diff
@@ -84,13 +85,7 @@
...
    "outputs": [],
    "source": [
-    "(\n",
-    "    model_id,\n",
-    "    model_version,\n",
-    ") = (\n",
-    "    \"meta-textgeneration-llama-2-7b-f\",\n",
-    "    \"*\",\n",
-    ")"
+    "model_id, model_version = \"meta-textgeneration-llama-2-7b-f\", \"*\""
```
Comment (ulrichkr) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/llama-2-chat-completion.ipynb:88]: please pin model versions

**Review by akrishna1995 — APPROVED** [review]
LGTM - can you please confirm that the datasets used is in the public s3 repo

**Comment by akrishna1995** [comment]
grammar check is failing, can you please verify

**Comment by akrishna1995** [comment]
partner team confirmed on slack that the grammar errors are from domain specific terms. Merging

**Pull request merged**